### ISSUE TITLE
ocropus-gpageseg --> too many connected compenents   / 

### ISSUE THREAD

**Issue by jmokoistinen** [tmbdev/ocropy#67]
This is my problem when trying to ocr with my pngs

book/0001.bin.png SKIPPED too many connnected components for a page image (21112 > 1176) (use -n to disable this check)

#####

./ocropus-gpageseg -n book/0001.bin.png
INFO:  
INFO:  book/0001.bin.png
INFO:  scale 4.472135955
Traceback (most recent call last):
  File "./ocropus-gpageseg", line 423, in safe_process1
    process1(job)
  File "./ocropus-gpageseg", line 373, in process1
    print_error("%s: scale (%g) less than --minscale; skipping\n"%(fname,str(scale)))
TypeError: float argument required, not str

**Comment by Yenthe666** [comment]
@jmokoistinen when using -n behind the command it'll be skipped and the script will work.
-n disables all checks.

**Comment by rajeevkannav** [comment]
`./ocropus-gpageseg '/home/rajeev/workspace/ocr-py/shots-1.jpg' -n`

> INFO:  
> INFO:  ########## ./ocropus-gpageseg /home/rajeev/workspace/ocr-py/shots-1.jpg
> INFO:  
> INFO:  /home/rajeev/workspace/ocr-py/shots-1.jpg
> INFO:  scale 8.0
> Traceback (most recent call last):
>   File "./ocropus-gpageseg", line 423, in safe_process1
>     process1(job)
>   File "./ocropus-gpageseg", line 373, in process1
>     print_error("%s: scale (%g) less than --minscale; skipping\n"%(fname,str(scale)))
> TypeError: float argument required, not str

**Comment by zuphilip** [comment]
The initial error
```
File "./ocropus-gpageseg", line 373, in process1
print_error("%s: scale (%g) less than --minscale; skipping\n"%(fname,str(scale)))
TypeError: float argument required, not str
```
is corrected by 1c06ec3.

This will now produce an error message for the user, because the computed `scale` does not seem to be correct, because it is too small (maybe tiny font size? newspaper?). As indicated by @rajeevkannav you can overwrite the default values for these checks.
